### ISSUE TITLE
Company / Channels / Publications

### ISSUE THREAD

**Issue by dom-advancedcommerce** [nozzlegear/ShopifySharp#1075]
We are tend to focus on the standard data (online channel for data) and we seeing more and more customers starting to use Markets for pricing.. etc. This makes things more challenging forcing us to move more of our code to GrpahQL... More recently we have been asked to look at B2B catalogues pricing and publishing.

It is all a bit disconnected, between the data that can be retrieved via the Shopify REST API and the tangle of weeds using the Shopify Graph API.

It seems Shopify themselves have yet to connect catalogues to publications, therefore this might be a mute point atm, but 
I was wondering if and how far ShopifySharp have delved into this and any Graph queries your using to retrieve publication data by company b2b catalogue publications verses channel publications?

**Comment by nozzlegear** [comment]
Yes, and I hope to write a big guide for using the GraphService soon! I've got a half formed draft in my backlog right now.

**Comment by czupet** [comment]
I would pay for that!
